src/master-playlist-controller.js

```diff
@@ -105,6 +106,29 @@ const getCodecs = function(media) {
   return defaultCodecs;
 };
 
+const audioProfileFromDefault = (master, audioGroupId) => {
```

Comment:
Maybe this would be a good opportunity to move this and other codec related helper functions into `util/codecs.js`?